compiler/rustc_session/src/options.rs

```diff
@@ -2331,6 +2331,8 @@ options! {
         "the size at which the `large_assignments` lint starts to be emitted"),
     mutable_noalias: bool = (true, parse_bool, [TRACKED],
         "emit noalias metadata for mutable references (default: yes)"),
+    namespaced_crates: bool = (false, parse_bool, [TRACKED],
+        "allow crates to be nested in other crates (default: no)"),
```

Comment:
Since you renamed `nested_crates` to `namespaced` crates, perhaps you also meant to update this description?

```suggestion
        "allow crates to be namespaced by other crates (default: no)"),
```